src/runtime/vdom/vdom-render.ts

```diff
@@ -737,18 +756,22 @@ const relocateSlotContent = (elm: d.RenderNode) => {
             }
 
             if (node['s-sr']) {
+              // is `node` itself a slot reference (as `childNode` is) ?
```

Comment:
Minor - unaddressed TODO(?). This reads a bit more as a question rather than an implementation not. Unsure if that was the intent or not here

Reply:
I was describing the check here, but I don't think it adds _so_ much so I'll just remove this line